manifest.yaml

```diff
@@ -195,3 +201,27 @@ actions:
         placeholder: password
         nullable: false
         default: ""
+  add-watchtower:
+    name: "Add a watchtower to your LND Node"
+    description: "Register a watchtower to use for future sessions/backups"
+    warning: "Make sure that the target LND node has watchtowers enabled!"
```

Comment:
Isn't the "target LND node" a totally different pubkey from the node's pubkey anyway?

Reply:
Yes it is. how does that negate this warning?

Reply:
Presumably there'd be no way for the user to connect to a watchtower using a non-watchtower pubkey?

Reply:
they could enter the node pubkey instead, and lnd will accept it without giving an error. that's why its important to highlight in the docs that they choose the correct URI

Reply:
Cool, perhaps update this warning with that info then? Seems a bit misleading to ask the user to "make sure the target LND node has watchtowers enabled". What the user really wants to do is "make sure the target is a watchtower pubkey, rather than an LND node pubkey"

Reply:
added.